buzzmobile/sense/inputer/inputer.py

```diff
@@ -10,7 +10,7 @@ def input_node():
     # Publish a nonsense initial value to make sure subscribers don't explode
     pub.publish("")
     while not rospy.is_shutdown():
-        new_dst = raw_input("Dest >> ")
+        new_dst = input("Dest >> ")
```

Comment:
This is only advisable in python3, correct? Can you make sure that when we do `rosnode buzzmobile inputer` ros will actually use py3 and not py2?